src/notifications/AlertBox.scss

```diff
@@ -46,6 +49,14 @@
       color: var(--bloom-color-white);
       text-decoration: underline;
     }
+
+    svg {
+      fill: var(--bloom-color-white);
+    }
+
+    .close {
+      color: var(--bloom-color-white);
+    }
```

Comment:
I'm not seeing this class get added anywhere. Looks like "text-white" class is added to the close button when it is inverted

Reply:
I removed the tailwind class and added a custom one, thank you!